routers/root.py

```
@@ -548,12 +556,36 @@ def chat_route(
     except (IndexError, BotIntegration.DoesNotExist):
         raise HTTPException(status_code=404)
 
+    if conversation_id:
+        try:
+            conversation: Conversation = Conversation.objects.get(
+                id=api_hashids.decode(conversation_id)[0],
+            )
+        except (IndexError, Conversation.DoesNotExist):
+            raise HTTPException(status_code=404)
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)